www/includes/easyparliament/templates/html/mp/votes.php

```diff
@@ -85,13 +85,17 @@
                         $party_position = $key_vote['party_position'] ;
                         $comparison_party = $data["comparison_party"];
                         $current_party_comparison = $data["current_party_comparison"];
-                        $party_voting_line = sprintf("%s, %s", $party, $diff['party_voting_summary']);
+                        $unslugified_comparison_party = ucwords(str_replace('-', ' ', $comparison_party));
```

Comment:
Feels like we have a lookup somewhere for this, but can't find it.

Reply:
It's in principle in people.json but I couldn't see it was made available anywhere. Nothing says this approach *has* to work, but does seem to for parties we care about.